plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/index.tsx

```diff
@@ -40,6 +42,14 @@ if ( shouldRegisterBlock ) {
 	// Select Simple product type
 	dispatch( woocommerceTemplateStateStore ).switchProductType( 'simple' );
 
+	// Extend editor, blocks, etc
```

Comment:
I guess this comment could be a bit more precise too? Something like:

```suggestion
	// Register a plugin that adds a product type selector to the template sidebar.
```